config/rbac/role.yaml

```diff
@@ -56,6 +56,21 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - get
+  - list
+  - patch
+  - watch
+- apiGroups:
+  - "storage.k8s.io"
+  resources:
+  - storageclasses
+  verbs:
+  - get
```

Comment:
Pls, add the following in above of the `func (r *EtcdClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error)` (https://github.com/aenix-io/etcd-operator/blob/main/internal/controller/etcdcluster_controller.go#L72) to generate `role.yaml`:

```go
// +kubebuilder:rbac:groups="",resources=persistentvolumeclaims,verbs=get;list;patch;watch
// +kubebuilder:rbac:groups=storage.k8s.io,resources=storageclasses,verbs=get
```

Otherwise it will be overwritten by the next 'make manifests' run.

Reply:
sure, done